manifests/jetty.pp

```diff
@@ -110,6 +106,13 @@
     mode    => '0644',
     recurse => false,
     notify  => Class['shibboleth_idp::service'],
+  } ->
+
+  file { $idp_jetty_log_dir:
+    ensure => directory,
+    owner  => $shibboleth_idp::shib_user,
+    group  => $shibboleth_idp::shib_group,
+    mode   => '0644',
```

Comment:
Is this a good mode? How about 0750?
This seems to have the same issue... A file mode on a dir. Which probably works, but maybe not the best. and also it may be nice to block the world and only allow the jetty user/group
```
file { $jetty_files:
```